README.md

```diff
@@ -2,30 +2,76 @@
 ![Version](https://img.shields.io/github/v/tag/OpenMined/PyDP?color=green&label=pypi)
 ![License](https://img.shields.io/github/license/OpenMined/PyDP)
 
+## Introduction to PyDP
 
-# PyDP
-
-In today's data-driven world, data analytics is used by researchers or data scientists to create better models or innovative solutions for a better future. These models often tend to handle sensitive or personal data, which brings in some privacy concerns. For example, some AI models can memorize details about the data they've trained on and could leak these details later on. Differential privacy is a mathematical framework for measuring this privacy leakage and reducing the possibility of it happening. 
+In today's data-driven world, data analytics is used by researchers or data scientists to create better models or innovative solutions for a better future. These models often tend to handle sensitive or personal data, which brings in some privacy concerns. For example, some AI models can memorize details about the data they've trained on and could leak these details later on. Differential privacy is a mathematical framework for measuring this privacy leakage and reducing the possibility of it happening.
 
 This is where PyDP comes in. PyDP is a Python wrapper for Google's [Differential Privacy](https://github.com/google/differential-privacy) project. The library provides a set of ε-differentially private algorithms, which can be used to produce aggregate statistics over numeric data sets containing private or sensitive information. Thus, PyDP is helping us achieve better privacy.
 
+**Brief introduction to google's Differential privacy project :-** 
+
+The Differentia Privacy project of google contains a set of libraries of ε- and (ε, δ)-differentially private algorithms, which can be used to produce aggregate statistics over numeric data sets containing private or sensitive information. The functionality is currently available in C++, Go and Java.
+
+Currently, it provides algorithms to compute the following:
```

Comment:
Can we remove this table?

Reply:
Yes we can remove it, I added it to add a brief intro to google's DP project.